### DISCUSSION TITLE
How to bind an input component to the signal store state property using two-way data binding

### DISCUSSION THREAD

**Discussion by caboodal** [ngrx/platform#4431] [Q&A]
I've been trying to workout how I bind a search text property in my state to an input component using two-way data binding. If I have an input component on my page how do I bind that to a signal store property so that I can subsequently use it in my computed method that returns a list of filtered rows?

What am I missing? Should I simply be using a JS onchange event and then update the store search property?

**Comment by AStoker** [comment]
Have you figured this one out by chance?
